### PR TITLE
git-clone, git-pull: add Chinese Translation

### PR DESCRIPTION
**Pull request by kasodesyn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kasodesyn** [comment]
Hello everyone, Salty Superman is here again! pr on bed.... 🛏️ zzz~~~

File: pages.zh/common/git-pull.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# git pull
+
+> 从远程代码库拉取分支，并将其合并到本地代码库。
+> 更多信息: <https://git-scm.com/docs/git-pull>.
```
Comment (navarroaxel) [inline pages.zh/common/git-pull.md:4]:
```suggestion
> 更多信息：<https://git-scm.com/docs/git-pull>.
```

File: pages.zh/common/git-clone.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# git clone
+
+> 克隆现有的代码库。
+> 更多信息: <https://git-scm.com/docs/git-clone>.
```
Comment (navarroaxel) [inline pages.zh/common/git-clone.md:4]:
```suggestion
> 更多信息：<https://git-scm.com/docs/git-clone>.
```

File: pages.zh/common/git-clone.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- 克隆一个现有的代码库，只获取默认分支上10个最新的提交（对节省时间很有用）：
+
+`git clone --depth {{10}} {{远程代码库地址}}`
+
+- 克隆一个现有的、特定远程分支的代码库：
+
+`git clone --branch {{分支名称}} --single-branch {{远程代码库地址}}`
+
+- 使用SSH命令克隆一个现有的代码库：
```
Comment (blueskyson) [inline pages.zh/common/git-clone.md:34]:
```suggestion
- 使用 SSH 命令克隆一个现有的代码库：
```

**Review by blueskyson — APPROVED** [review]

**Comment by kasodesyn** [comment]
Thanks to review, this is my second time submitting a pull request and I'm slowly finding some sense. I'd be happy to be part of the review team if needed, and thanks again for providing such a great tool!

@navarroaxel  @blueskyson

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**